src/components/textfield/textfield.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../styles/scss/base/variables.scss';
```

Comment:
No need to use the extra `../../` instead `@import "~theme/base/variables";`

Reply:
could easily see after pull from upstream from development